dessert/bûche de noël

```diff
@@ -0,0 +1,39 @@
+RECETTE BUCHE DE NOEL AU BEURRE
```

Comment:
Maybe convert the file in markdown format and rename the file to `buche-de-noel.md`